packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.3
```

Comment:
New features are minor versions in semver.

Reply:
Fixed